.cursor/rules/tailwind_v4.md

```
@@ -0,0 +1,218 @@
+// Update globs depending on your framework
+---
+name: tailwind_v4
+description: Guide for using Tailwind CSS v4 instead of v3.x
+globs: ["**/*.{js,ts,jsx,tsx,mdx,css}"]
+tags:
+  - tailwind
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)